var/ramble/repos/builtin/modifiers/intel-aps/modifier.py

```diff
@@ -39,6 +55,33 @@ class IntelAps(BasicModifier):
         "mpi_command", " aps {aps_flags} ", method="append", modes=["mpi"]
     )
 
+    modifier_variable(
+        "aps_stat_level",
+        default="1",
+        description="Used to define the APS_STAT_LEVEL env variable",
+        mode="mpi",
+    )
+
+    modifier_variable(
```

Comment:
Can we generate a `values` argument using the keys from `_PREDEFINED_REPORTS`?

Reply:
This `aps_extra_reports` var allows for a comma-separated string that defines multiple report types, it also allows passing in "custom" report specs. Given that it doesn't have a static allow-list.

Reply:
Ah OK cool. Well can we maybe have it extract the "known" report types from the dict when generating the description string?

Reply:
Good point, updated.